content/klc/S6.2.adoc

```diff
@@ -1,9 +1,24 @@
 +++
-brief = "Component fields must be filled appropriately"
+brief = "Symbol and alias fields and metadata filled out as required"
 +++
 
 . *Reference* field is selected appropriately for the symbol and is _visible_
 . *Value* field contains the name of the symbol and is _visible_
-. *Footprint* field contains footprint link for atomic parts, and is empty for generic parts. It must be _invisible_
-. *Datasheet* field is left empty, and is _invisible_
-. The symbol contains no other custom fields
\ No newline at end of file
+. *Footprint* field contains footprint link for fully specified symbols, and is empty for generic symbols. It must be _invisible_
```

Comment:
Other bullets here would replace the `. It must be` with `and is`.

Reply:
Changed this section to be more readable